resources/builder.inc.sh

```diff
@@ -1049,9 +1156,39 @@ builder_parse() {
       target=
     fi
 
+    # Expand shorthand parameters
+
+    new_action=$(_builder_expand_shorthand $action "${_builder_actions[@]}") ||
+      case $? in
+        1)
+          action=$new_action
+          ;;
+        *)
+          builder_warn "Parameter $action has $? matches, could mean any of {$new_action}"
```

Comment:
Any reason you used builder_warn instead of builder_die? (The example in l.261 uses die)

Reply:
I just liked it in yellow rather than red :grin: -- it looks friendlier for this scenario.